### PR TITLE
whitelisting checks for BF

### PR DESCRIPTION
**Pull request by rajatparida86**
whitelisting checks for BF

### CODE REVIEW

File: src/test/java/de/zalando/zmon/dataservice/data/KairosDbStoreTest.java [hunk 1]
```diff
@@ -43,6 +44,9 @@
     @Autowired
     private DataServiceMetrics metrics;
 
+    @Autowired
+    private WhitelistedChecks whitelistedChecks;
```
Comment (bocytko) [inline src/test/java/de/zalando/zmon/dataservice/data/KairosDbStoreTest.java]: Where do you actually verify that the white listing works properly?

File: src/main/java/de/zalando/zmon/dataservice/config/WhitelistedChecks.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    public void updateWhitelist() {
+        try {
+            LOG.debug("started updating whitelist");
+            Request request = Request.Get(hostname + "/api/v1/entities/zmon-checkid-whitelist")
+                    .addHeader("Authorization", "Bearer " + wrapper.get());
+            String data = executor.execute(request).returnContent().toString();
+            JsonNode jsonNode = objectMapper.readTree(data);
+            Stream<Integer> checkIdsStream = jsonNode.findValues("check_ids").stream().map(JsonNode::intValue);
+            this.whitelist = checkIdsStream.collect(Collectors.toList());
+            LOG.debug("whitelist updated");
```
Comment (bocytko) [inline src/main/java/de/zalando/zmon/dataservice/config/WhitelistedChecks.java]: can you print the size before and after?

File: src/test/java/de/zalando/zmon/dataservice/data/KairosDbStoreTest.java (hunk 1 above)
Comment (alexkorotkikh) [reply src/test/java/de/zalando/zmon/dataservice/data/KairosDbStoreTest.java]: is it about adding tests or can you please explain what do you mean?

**Comment by avietrov** [comment]
Is usage of lists intentional (over sets with constant `.contains`)?

**Comment by rajatparida86** [comment]
👍

**Comment by sbrauner-z** [comment]
👍

**Comment by alexkorotkikh** [comment]
:+1:

**Pull request merged**